plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/ImplementStddev.java

```diff
@@ -0,0 +1,70 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc.expression;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.matching.Capture;
+import io.trino.matching.Captures;
+import io.trino.matching.Pattern;
+import io.trino.plugin.jdbc.JdbcColumnHandle;
+import io.trino.plugin.jdbc.JdbcExpression;
+import io.trino.spi.connector.AggregateFunction;
+import io.trino.spi.expression.Variable;
+import io.trino.spi.type.DoubleType;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.base.Verify.verify;
+import static com.google.common.base.Verify.verifyNotNull;
+import static io.trino.matching.Capture.newCapture;
+import static io.trino.plugin.jdbc.expression.AggregateFunctionPatterns.basicAggregation;
+import static io.trino.plugin.jdbc.expression.AggregateFunctionPatterns.expressionType;
+import static io.trino.plugin.jdbc.expression.AggregateFunctionPatterns.functionName;
+import static io.trino.plugin.jdbc.expression.AggregateFunctionPatterns.singleInput;
+import static io.trino.plugin.jdbc.expression.AggregateFunctionPatterns.variable;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static java.lang.String.format;
+
+public class ImplementStddev
+        implements AggregateFunctionRule
+{
+    private static final List<String> STDDEV_FUNCTION_NAMES = ImmutableList.of("stddev", "stddev_samp");
+    private static final Capture<Variable> INPUT = newCapture();
+
+    @Override
+    public Pattern<AggregateFunction> getPattern()
+    {
+        return basicAggregation()
+                .with(functionName().matching(name -> STDDEV_FUNCTION_NAMES.stream().anyMatch(name::equalsIgnoreCase)))
+                .with(singleInput().matching(
+                        variable()
+                                .with(expressionType().matching(DoubleType.class::isInstance))
+                                .capturedAs(INPUT)));
+    }
+
+    @Override
+    public Optional<JdbcExpression> rewrite(AggregateFunction aggregateFunction, Captures captures, RewriteContext context)
+    {
+        Variable input = captures.get(INPUT);
+        JdbcColumnHandle columnHandle = (JdbcColumnHandle) context.getAssignment(input.getName());
+        verifyNotNull(columnHandle, "Unbound variable: %s", input);
+        verify(columnHandle.getColumnType().equals(DOUBLE));
+        verify(aggregateFunction.getOutputType().equals(DOUBLE));
+
+        return Optional.of(new JdbcExpression(
+                format("stddev_samp(%s)", columnHandle.toSqlExpression(context.getIdentifierQuote())),
```

Comment:
can you pleasee rebase on top of #6667 ?
this line would need to be changed to

```suggestion
                format("stddev_samp(%s)", context.getIdentifierQuote().apply(columnHandle.getColumnName())),
```